src/Content/Text/BBCode.php

```diff
@@ -1247,16 +1247,28 @@ private static function removePictureLinksCallback($match)
 		return $text;
 	}
 
-	private static function expandLinksCallback($match)
+	/**
+	 * Callback: Expands links from given $match array
```

Comment:
Same comment as the parameter documentation earlier, this is an exact duplicate of the method signature which doesn't provide any more information.

Reply:
I normally document more than lesser. :)